### PR TITLE
Add coverage report

### PR DESCRIPTION
**Pull request by vvmruder**
Use Codecov to produce coverage report.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -279,7 +279,7 @@ test-postgis:
 
 .PHONY: test-core
 test-core: .venv/requirements-timestamp
-	$(VENV_BIN)/py.test -vv $(PYTEST_OPTS) --cov-config .coveragerc --cov $(PACKAGE) --cov-report term-missing:skip-covered tests/core
+	$(VENV_BIN)/py.test -vv $(PYTEST_OPTS) --cov-config .coveragerc --cov $(PACKAGE) --cov-report=xml term-missing:skip-covered tests/core
```
Comment (mki-c2c) [inline Makefile:282]: remove term-missing:skip-covered

File: .github/workflows/ci.yaml [hunk 2]
```diff
@@ -62,6 +62,18 @@ jobs:
...
           PYTHON_TEST_VERSION: ${{ matrix.python-version }}
         run: make test-core
+      - name: Upload coverage to Codecov
+        uses: codecov/codecov-action@v2
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
+          directory: ./coverage/reports/
+          env_vars: OS,PYTHON
+          fail_ci_if_error: true
+          files: ./coverage1.xml,./coverage2.xml
```
Comment (mki-c2c) [inline .github/workflows/ci.yaml:72]: ./coverage.xml only

**Review by mki-c2c — CHANGES_REQUESTED** [review]
CI not running yet

File: Makefile (hunk 1 above)
Comment (vvmruder) [reply Makefile]: thx

File: .github/workflows/ci.yaml (hunk 2 above)
Comment (vvmruder) [reply .github/workflows/ci.yaml]: thx

**Review by mki-c2c — APPROVED** [review]
looks nice, thanks, we now have a great tool.

I don't manage to click on the coverage map though, it shows me a page "Sorry, we couldn't find that page"

However, I think it's a good ides to merge (giving the coverage diff with master) and then see how we can live with it. Doing later fixes or additions/removal.

**Pull request merged**